MeetU/MeetU/Content/Create.less

```diff
@@ -0,0 +1,7 @@
+﻿.form-group .help-block {
```

Comment:
it will affect other styles. if you intend to affect all .form-group, you need to extract them to another file. if it would be only working in create page, you should add more restriction to these styles.